README.md

```diff
@@ -12,7 +12,7 @@ DKPro Argumentation is available from Maven Central. For working with the UIMA t
 <dependency>
     <groupId>de.tudarmstadt.ukp.dkpro.argumentation.types</groupId>
     <artifactId>de.tudarmstadt.ukp.dkpro.argumentation.types</artifactId>
-    <version>0.0.2</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
I'd intentionally leave it to 0.0.2 as this version is on Maven Central. This README.md is the only documentation for the current version. Solution: add a note about the stable version 0.0.2, don't change the guidelines for Maven Central dependency.